src/Events/Services/LabTransmitEvent.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ *   package   OpenEMR
+ *   link      http://www.open-emr.org
+ *  author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  Copyright (c)
+ *  All rights reserved
+ *
+ */
+
+namespace OpenEMR\Events\Services;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+class LabTransmitEvent extends Event
+{
+    /**
+     * This event is triggered when a lab is to be transmitted
+     */
+    const EVENT_LAB_TRANSMIT = 'lab.transmit';
+
+    /**
+     * This event is triggered when a lab requisition form is returned from Quest
+     * Requisition form has to be enabled in the globals
+     */
+    const EVENT_LAB_POST_ORDER_LOAD = 'lab.post_order_load';
+
+    private string $order;
+    public function __construct($hl7)
```

Comment:
If this is going to be a generic event do you think that every lab transmission will use an HL7 string as the order transmission?  Just wondering if we need to consider options where more data needs to be sent as part of the lab transmit event.  If it will always be a string then this is fine.

Reply:
@adunsulag all of my dealings with labs. They all have used HL7 string. I believe until a new standard is developed on high. It will always be a string. IMO

Reply:
Sounds good.